docs/api/commands/exec.md

```diff
@@ -44,7 +45,9 @@ This will create a new Bash session in the container `ubuntu_bash`.
 
 ## Divergence
 
-There is no known divergence between the Triton SDC Docker and Docker Inc. implementations of this method. Please contact Joyent support or file a ticket if you discover any.
+There is no known divergence between the Triton SDC Docker and Docker Inc.
+implementations of this method. Please contact Joyent support or file a ticket
```

Comment:
@misterbisson you want this one paragraph formatted differently than the rest of the file? I'd highly recommend against changing the format from upstream.